great_expectations/cli/cli.py

```diff
@@ -350,28 +350,22 @@ def profile(datasource_name, data_assets, profile_all_data_assets, directory, ba
 )
 @click.option('--site_name', '-s',
               help='The site for which to generate documentation. See data_docs section in great_expectations.yml')
-@click.option('--data_asset_name', '-dan',
-              help='The data asset for which to generate documentation. Must also specify --site_name.')
 @click.option(
     "--view/--no-view",
     help="By default open in browser unless you specify the --no-view flag",
     default=True
 )
-def build_docs(directory, site_name, data_asset_name, view=True):
+def build_docs(directory, site_name, view=True):
```

Comment:
Can we allow specification of a resource identifier here?
I'm thinking along these lines:
```
great_expectations build-docs --data_asset_name my_datasource/my_generator/my_generator_asset --expectation_suite_name my_suite --run_id blarg
```

Essentially, I'm thinking we should expose the same marginal build functionality that the action is using via the CLI.

Reply:
The CLI was where the original issue was identified. This needs to function here.

Reply:
Based on our call, we might consider this in future, but not for this release. Marking as resolved.